plugins/externalwrench/include/gazebo/ApplyExternalWrench.hh

```diff
@@ -8,6 +8,7 @@
 #include <gazebo/physics/Link.hh>
 #include <gazebo/physics/World.hh>
 #include <gazebo/physics/Model.hh>
+#include <gazebo/physics/physics.hh>
```

Comment:
These catch-it-all headers are tipically a huge reason behind the big compilation time. Furthermore, it does not seems that you added any new class to use in this PR, so why was this header added?

Reply:
@traversaro this was added to retrieve the physics engine https://github.com/Yeshasvitvs/gazebo-yarp-plugins/blob/feature/externalwrench-smoothing/plugins/externalwrench/src/ApplyExternalWrench.cc#L62

Reply:
Ack. Anyhow, if it is used in `.cc`, why it is not included just in the `.cc` file?

Reply:
@traversaro changed the header file to be more specific `PhysicsTypes.hh` than the catch-it-all `Physics.hh` and moved it to the `.cc` file https://github.com/robotology/gazebo-yarp-plugins/pull/430/commits/7e3e398722ecddfde352425ce29ad25a5ed0cdd6